config/core/resources/sinkbindings.yaml

```diff
@@ -26,7 +26,7 @@ spec:
   group: sources.knative.dev
   versions:
     - &version
-      name: v1alpha1
+      name: v1beta1
```

Comment:
reordering versions has been problematic in the past, does it now work?

Reply:
oops, I wasn't aware of the problem. Let me revert the reordering just in case.
Thanks for pointing that out